### PR TITLE
[Bulk]Validation-Missing-Alltext6

### PR DESCRIPTION
**Pull request by Roudabeh1348**
This PR fixes 89 issues from missing alt text file from iis-docs.

### CODE REVIEW

File: iis/configuration/system.webServer/security/requestFiltering/verbs/add.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ The default installation of IIS 7 and later includes the Request Filtering role
 2. In **Server Manager**, click the **Manage** menu, and then click **Add Roles and Features**.
 3. In the **Add Roles and Features** wizard, click **Next**. Select the installation type and click **Next**. Select the destination server and click **Next**.
 4. On the **Server Roles** page, expand **Web Server (IIS)**, expand **Web Server**, expand **Security**, and then select **Request Filtering**. Click **Next**.  
-    [![](add/_static/image2.png)](add/_static/image1.png) .
+    [![Screenshot of Web Server I I S nad Security pane expanded with Request Filtering selected.](add/_static/image2.png)](add/_static/image1.png) .
```
Comment (RedZephyr13) [inline iis/configuration/system.webServer/security/requestFiltering/verbs/add.md:49]: Small typo for "and."

File: iis/configuration/system.applicationHost/sites/site/ftpServer/security/authentication/customAuthentication/providers/index.md [hunk 2]
```diff
@@ -140,11 +140,11 @@ To support FTP publishing for your Web server, you must install the FTP service.
...
 3. In the site's **Home** pane, double-click the **FTP Authentication** feature.
 4. When the **FTP Authentication** page displays, click **Custom Providers...** in the **Actions** pane.  
-    [![](index/_static/image10.png)](index/_static/image9.png)
+    [![Screenshot of F T P Authentication page displaying Custom Providers tab in Actions pane.](index/_static/image10.png)](index/_static/image9.png)
 5. When the **Custom Providers** dialog box displays, click to select **IIS Manager Authentication**. Click **OK**.  
-    [![](index/_static/image12.png)](index/_static/image11.png)
+    [![Screenshot of Custom Providers dialog box displaying I I S Manager Authentication option.](index/_static/image12.png)](index/_static/image11.png)
 6. Your **FTP Authentication** page should now show **IIS Manager Authentication** enabled.  
-    [![](index/_static/image14.png)](index/_static/image13.png)
+    [![Screenshot of F T P AUthentication page showing I I S Manager Authentication feature enabled.](index/_static/image14.png)](index/_static/image13.png)
```
Comment (RedZephyr13) [inline iis/configuration/system.applicationHost/sites/site/ftpServer/security/authentication/customAuthentication/providers/index.md:147]: Small typo on "Authentication."

File: iis/configuration/system.webServer/security/requestFiltering/filteringRules/filteringRule/scanHeaders/index.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ The default installation of IIS 7 and later includes the Request Filtering role
 2. In **Server Manager**, click the **Manage** menu, and then click **Add Roles and Features**.
 3. In the **Add Roles and Features** wizard, click **Next**. Select the installation type and click **Next**. Select the destination server and click **Next**.
 4. On the **Server Roles** page, expand **Web Server (IIS)**, expand **Web Server**, expand **Security**, and then select **Request Filtering**. Click **Next**.  
-    [![](index/_static/image2.png)](index/_static/image1.png) .
+    [![Screenshot of Server Role page displaying the Web Server I I I S and Security node expanded and Request Filtering selected.](index/_static/image2.png)](index/_static/image1.png) .
```
Comment (RedZephyr13) [inline iis/configuration/system.webServer/security/requestFiltering/filteringRules/filteringRule/scanHeaders/index.md:40]: One too many I's for "I I S."

File: iis/configuration/system.webServer/security/requestFiltering/filteringRules/filteringRule/scanHeaders/index.md [hunk 4]
```diff
@@ -67,7 +67,7 @@ The default installation of IIS 7 and later includes the Request Filtering role
 2. In **Control Panel**, click **Programs and Features**, and then click **Turn Windows Features on or off**.
 3. Expand **Internet Information Services**, then **World Wide Web Services**, and then **Security**.
 4. Select **Request Filtering**, and then click **OK**.   
-    [![](index/_static/image8.png)](index/_static/image7.png)
+    [![Screenshot shows that the Internet Information Services and Security node expanded. Request Filtering is highlighted.](index/_static/image8.png)](index/_static/image7.png)
```
Comment (RedZephyr13) [inline iis/configuration/system.webServer/security/requestFiltering/filteringRules/filteringRule/scanHeaders/index.md:70]: Maybe, "Screenshot that shows the Internet Information Services and Security node expanded."

File: iis/configuration/system.webServer/security/requestFiltering/verbs/add.md (hunk 1 above)
Comment (Roudabeh1348) [reply iis/configuration/system.webServer/security/requestFiltering/verbs/add.md:49]: fixed.

File: iis/configuration/system.applicationHost/sites/site/ftpServer/security/authentication/customAuthentication/providers/index.md (hunk 2 above)
Comment (Roudabeh1348) [reply iis/configuration/system.applicationHost/sites/site/ftpServer/security/authentication/customAuthentication/providers/index.md:147]: fixed.

**Comment by Roudabeh1348** [comment]
@Rick-Anderson @mcleblanc This PR fixes validation warnings/suggestions per a platform-wide push to increase content health scores. Please review at your convenience and sign off or merge if you approve or let me know if I need to make any other changes. cc: @MonicaRush @snoviking

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**